pkg/kube/apis/quarksstatefulset/v1alpha1/types.go

```diff
@@ -29,6 +30,8 @@ var (
 	LabelPodOrdinal = fmt.Sprintf("%s/pod-ordinal", apis.GroupName)
 	// LabelQStsName is the name of the QuarksStatefulSet owns this resource
 	LabelQStsName = fmt.Sprintf("%s/quarks-statefulset-name", apis.GroupName)
+	// LabelActiveContainer is the active container on an active/passive setup
+	LabelActiveContainer = fmt.Sprintf("%s/pod-designation", apis.GroupName)
```

Comment:
Designation is an ambiguous term - it can also mean "the name of something" 
I think `pod-active` makes more sense.